src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -735,7 +735,7 @@ private void CheckSwapOrPageFileAndRaiseNotification()
                     return;
 
                 var swapSize = MemoryInformation.GetMemoryInfo().TotalSwapSize;
-                if (swapSize < Configuration.PerformanceHints.MinSwapSize)
+                if (swapSize < Configuration.PerformanceHints.MinSwapSize - new Sparrow.Size(100, SizeUnit.Megabytes))
```

Comment:
Let's change to 128 and introduce a variable called 'errorThreshold' or sth like that.

Also there is a fialing test.

Reply:
Do you want the variable outside the function (static or field)?

Reply:
Just variable.
```
var swapSize = MemoryInformation.GetMemoryInfo().TotalSwapSize;
var errorThreshold = new Sparrow.Size(128, SizeUnit.Megabytes);
if (swapSize < Configuration.PerformanceHints.MinSwapSize - errorThreshold )
```